### PR TITLE
update vcenter_ovf_libraryitem docs

### PR DESCRIPTION
**Pull request by mikemorency**
##### SUMMARY
Related to the discussion in https://github.com/ansible-collections/vmware.vmware_rest/issues/385

This change updates the module documentation to mention that template metadata is not updated as expected. It also includes a workaround so users can update the template description if needed.

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
vcenter_ovf_libraryitem

### CODE REVIEW

File: plugins/modules/vcenter_ovf_libraryitem.py [hunk 1]
```diff
@@ -261,93 +265,85 @@
...
 
-- name: Create a new VM from the OVF and specify the host and folder
+- name: Update The Template Description
   vmware.vmware_rest.vcenter_ovf_libraryitem:
-    ovf_library_item_id: '{{ (lib_items.value|selectattr("name", "equalto", "golden_image")|first).id }}'
-    state: deploy
+    session_timeout: 2900
+    source:
+      type: VirtualMachine
+      id: "{{ lookup('vmware.vmware_rest.vm_moid', vm_name) }}"
```
Comment (anna-savina) [inline plugins/modules/vcenter_ovf_libraryitem.py:309]: for consistency
```suggestion
      id: "{{ lookup('vmware.vmware_rest.vm_moid', 'My VM') }}"
```

File: plugins/modules/vcenter_ovf_libraryitem.py [hunk 2]
```diff
@@ -261,93 +265,85 @@
...
     target:
-      resource_pool_id: "{{ lookup('vmware.vmware_rest.resource_pool_moid', '/my_dc/host/my_cluster/Resources') }}"
-      folder_id: "{{ lookup('vmware.vmware_rest.folder_moid', '/my_dc/vm') }}"
-      host_id: "{{ lookup('vmware.vmware_rest.host_moid', '/my_dc/host/my_cluster/esxi1.test/test_vm1') }}"
-    deployment_spec:
-      name: my_vm_from_ovf_on_a_host
-      accept_all_EULA: true
-      storage_provisioning: thin
+      library_id: "{{ library_id }}"
```
Comment (anna-savina) [inline plugins/modules/vcenter_ovf_libraryitem.py:311]: for consistency
```suggestion
      library_id: 1111111-1111111-1111111-111111
```

File: plugins/modules/vcenter_ovf_libraryitem.py [hunk 3]
```diff
@@ -261,93 +265,85 @@
...
+      library_item_id: "{{ _ovf_item.id }}"
```
Comment (anna-savina) [inline plugins/modules/vcenter_ovf_libraryitem.py:312]: according to the real result
```suggestion
      library_item_id: "{{ _ovf_item.ovf_library_item_id }}"
```

File: plugins/modules/vcenter_ovf_libraryitem.py [hunk 4]
```diff
@@ -261,93 +265,85 @@
...
-    error:
-      errors: []
-      information: []
-      warnings: []
-    resource_id:
-      id: vm-1078
-      type: VirtualMachine
-    succeeded: 1
   type: dict
+  sample:
```
Comment (anna-savina) [inline plugins/modules/vcenter_ovf_libraryitem.py:331]: I got this result with `vmware.vmware.content_library_item_info` module, but `vmware.vmware_rest.vcenter_ovf_libraryitem` module returns:
```
    "_ovf_item": {
        "ansible_facts": {
            "discovered_interpreter_python": "/usr/bin/python3"
        },
        "changed": true,
        "failed": false,
        "value": {
            "error": {
                "errors": [],
                "information": [],
                "warnings": []
            },
            "ovf_library_item_id": "ea695665-307d-44da-b813-398969fc3903",
            "succeeded": true
        }
    }
```

File: plugins/modules/vcenter_ovf_libraryitem.py (hunk 3 above)
Comment (mikemorency) [reply plugins/modules/vcenter_ovf_libraryitem.py:312]: `ovf_library_item_id` is only returned if the item already exists. If the item does not exist, the `id` attribute is returned. So i think the original is correct in this example

File: plugins/modules/vcenter_ovf_libraryitem.py (hunk 4 above)
Comment (anna-savina) [reply plugins/modules/vcenter_ovf_libraryitem.py:331]: hm... Are you sure? Update task gave me the same result as Create task. Can you check it please one more time

Comment (mikemorency) [reply plugins/modules/vcenter_ovf_libraryitem.py:331]: I updated the sample block. Hopefully its clearer that there are two possible return values

**Review by anna-savina — APPROVED** [review]
LGTM

**Pull request merged**